CapabilityStatement/CapabilityStatement-nhsdigital.json

```diff
@@ -33,59 +34,59 @@
             "resource": [
                 {
                     "type": "Account",
-                    "profile": "https://fhir.nhs.uk/StructureDefinition/NHSDigital-Account"
+                    "profile": "https://fhir.nhs.uk/StructureDefinition/England-Account"
```

Comment:
Not NHSEngland?  England seems a bit broad...  Same for all following instances

Reply:
DA decision was: https://fhir.nhs.uk/ResourceType/England-NAME